server/lobbyconnection.py

```diff
@@ -296,6 +299,12 @@ def send_updated_achievements(self, updated_achievements):
                                         self._logger.warning('Tried to ban player with invalid period')
                                         raise ClientError(f"Period '{period}' is not allowed!")
 
+                                    self._logger.warning(">>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>")
```

Comment:
This looks like left over stuff from when you were getting tests to work. Please delete

Reply:
😅